pkg/plugins/golang/v3/webhook.go

```diff
@@ -123,7 +127,13 @@ func (p *createWebhookSubcommand) InjectResource(res *resource.Resource) error {
 }
 
 func (p *createWebhookSubcommand) Scaffold(fs machinery.Filesystem) error {
-	scaffolder := scaffolds.NewWebhookScaffolder(p.config, *p.resource, p.force)
+	scaffolder := scaffolds.NewWebhookScaffolder(
+		p.config,
+		*p.resource,
+		p.force,
+		p.options.DoHubScaffold,
+		p.options.DoSpokeScaffold,
```

Comment:
We do not pass DoCoversion, for example. Could we try to me those works exactly the same way that we work with the other options?